src/engine/engine.rs

```diff
@@ -2,14 +2,16 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+extern crate devicemapper;
```

Comment:
I'm surprised this is necessary, since it is externeded in ```src/lib.rs```. It can probably be removed.